src/solc_0.8/defi/ERC20RewardPool.sol

```diff
@@ -173,7 +173,7 @@ contract ERC20RewardPool is
         uint256 recoverAmount;
 
         if (rewardToken == _stakeToken) {
-            recoverAmount = rewardToken.balanceOf(address(this)) - _totalSupply;
+            recoverAmount = _stakeToken.balanceOf(address(this)) - _totalSupply;
```

Comment:
I think we should change _stakeToken to rewardToken in getRewardsAvailable instead

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/835/commits/aa7d8726a41837a75d28132985f595da5014ccad